docs/ref/files/storage.txt

```diff
@@ -4,28 +4,43 @@ File storage API
 
 .. module:: django.core.files.storage
 
-Getting the current storage class
+Getting the default storage class
 =================================
 
-Django provides two convenient ways to access the current storage class:
+Django provides convenient ways to access the default storage class:
+
+.. data:: storages
+
+    The ``default`` key of ``django.core.files.storages`` provides access
+    to the default storage as defined by :setting:`STORAGES`.
 
 .. class:: DefaultStorage
 
     :class:`~django.core.files.storage.DefaultStorage` provides
-    lazy access to the current default storage system as defined by
-    :setting:`DEFAULT_FILE_STORAGE`. :class:`DefaultStorage` uses
-    :func:`~django.core.files.storage.get_storage_class` internally.
+    lazy access to the default storage system as defined by ``default`` key in
+    :setting:`STORAGES`. :class:`DefaultStorage` uses
+    :data:`~django.core.files.storage.storages` internally.
+
+.. data:: default_storage
```

Comment:
`default_storage` exists without this patch, we should document it in a separate PR.

Reply:
Reverted but I think it should stay because this pr changes the documented behavior.

Reply:
> Reverted but I think it should stay because this pr changes the documented behavior.

So let's document the current behavior in a separate patch (that can be backported) and then we can update the docs here.

Reply:
Here's a separate pr https://github.com/django/django/pull/15942